index.html

```diff
@@ -20,377 +20,273 @@
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="four-tags">
-          <div class="four-tag">
-           <span>Html</span>
+  
+        <div class="card-2">
+          <div class="fourth-card">
+            <div class="fourth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-4">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag2">
-           <span>bootstrap</span>
+          <div class="fifth-card">
+            <div class="fifth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-5">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag3">
-           <span>Ruby</span>
+          <div class="sixth-card">
+            <div class="sixth-block">
+              <h1>Website Portfolio</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-6">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
-
-
-
-
-        
-
-      </section>
-
-      
+      </div>
 
     </section>
-    <section class="about">
-      <h1>
-        About Me
-      </h1>
 
-      
-      <span>
-          Hello I’m a software developer! I can help you build a product , 
+    <section class="about">
+      <div class="about-me">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , 
           feature or website Look through some of my work and experience! 
           If you like what you see and have a project you need coded, 
           don’t hestiate to contact me.
-      </span>
+        </p>
 
-      <div class="about-resume">
-        <small>
-          Get My Resume
-        </small>
+        <div class="about-btn">
+          <h4>Get My Resume</h4>
+        </div>
```

Comment:
- [ ] This `Get my Resume` is the buttons, I would suggest you please use the button tag or `a` tag for better semantics HTML.
- [ ] As per the project requirement, please implement the `hovered`, and `Pressed functionality. See the image below for reference.
### Design
![Screenshot 2022-12-09 130627](https://user-images.githubusercontent.com/81727988/206647956-8fa0a027-1da5-42c4-ba03-8ba5a4c41fa5.png)